src/models/TimeTraveling.js

```diff
@@ -0,0 +1,216 @@
+import { observable } from "mobx"
+import { useComputed } from "./MobXHooks"
+import React, { useState, useRef } from "react"
+
+export const overrideDirectory = observable.box(null)
+global.ELECT_overrideDirectory = directory => overrideDirectory.set(directory)
+
+export function useDirectoryOverride() {
+  return [
+    useComputed(() => overrideDirectory.get(), []),
+    newDirectory => overrideDirectory.set(newDirectory),
+  ]
+}
+
+export function TimeMachine() {
+  const [override, setOverride] = useDirectoryOverride()
+  const [hover, setHover] = useState("")
+  const timeout = useRef()
+  return (
+    <div
+      data-active={override ? true : undefined}
+      css={{
+        textAlign: "center",
+        color: "#888",
+        "& > .status-text": { opacity: 0 },
+        "&:focus-within, &:focus, &:hover, &[data-active]": {
+          "& > .status-text": { opacity: 1 },
+        },
+      }}
+    >
+      <div className="status-text">
+        {hover && hover !== override ? (
+          <span>
+            <strong>Time machine:</strong> Time travel to {hover}
+          </span>
+        ) : override ? (
+          <span>
+            <strong>Time machine:</strong> Currently viewing {override}
+          </span>
+        ) : (
+          <span>
+            <strong>Time machine is inactive</strong> — viewing latest data
+          </span>
+        )}
+      </div>
+      <div css={{ display: "flex", height: 20 }}>
+        {dirs.slice(0, -1).map(([directory, time, voteCount], index) => (
+          <a
+            data-active={override === directory ? true : undefined}
+            href="javascript://"
+            css={{
+              background: "#eee",
+              "&:hover": { background: "#ccc" },
+              "&[data-active]": { background: "#888" },
+            }}
+            style={{
+              flex: `${Math.min(dirs[index + 1][1] - time, 1800)} 0 0px`,
+            }}
+            onClick={() =>
+              setOverride(directory === override ? null : directory)
```

Comment:
It would be better if we could introduce a style  to these <a>'s? For example, adding `border-right`

<img width="1328" alt="image" src="https://user-images.githubusercontent.com/1214890/54918100-f42a9780-4f2f-11e9-9aaa-af7064ff27eb.png">

Reply:
Borders added

Reply:
thanks!